api/axis.py

```diff
@@ -535,12 +535,14 @@ def __init__(self, _db, collection, api_class=Axis360API):
             Axis360BibliographicCoverageProvider(collection, api_class=self.api)
         )
 
-    def run_once(self, start, cutoff):
-        # Give us five minutes of overlap because it's very important
-        # we don't miss anything.
-        since = start-self.FIVE_MINUTES
+    def catch_up_from(self, start, cutoff, progress):
+        """Find Axis 360 books that changed recently.
+
+        :progress: A TimestampData representing the time previously
+        covered by this Monitor.
+        """
         count = 0
-        for bibliographic, circulation in self.api.recent_activity(since):
+        for bibliographic, circulation in self.api.recent_activity(start):
```

Comment:
This is typical of the simple changes made in this branch. A monitor becomes a `TimeLineMonitor`, its `run_once` becomes `catch_up_from`, and code for creating five minutes of overlap is removed, because `TimeLineMonitor` adds five minutes of overlap to the value it passes in to `start`.

Reply:
A `TimeLineMonitor` still has a `run_once` method. That still gets triggered or is it just `catch_up_from`?

Reply:
Yeah, the main entry point is `run`, which calls `run_once`, and then for a `TimelineMonitor` `run_once` calls `catch_up_from`.